RetrieveAVRISNGData.py

```diff
@@ -0,0 +1,40 @@
+from ftplib import FTP
```

Comment:
Please provide Apache Licence v2.0 license header as per all other files in this repository.